src/utils.ts

```diff
@@ -7,3 +7,32 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export const getArrOfItemsForCurrPage = (
+  sum: number,
+  sumOfPosPerPage: number,
```

Comment:
Consider renaming the variable **sumOfPosPerPage** to something more descriptive, such as **itemsPerPage**.
**sum** can also be **totalItems**